src/pages/productDetail.html

```diff
@@ -0,0 +1,407 @@
+                           <div class="product-info border-normal">
+                                <div class="product-name text-dark">
+                                    <h3 class="f-16 font-weight-bold">Cây Dạ Lam</h3>
+                                </div>
+                                <div class="product-rating">
+                                    <i class="fas fa-star"></i>
+                                    <i class="fas fa-star"></i>
+                                    <i class="fas fa-star"></i>
+                                    <i class="fas fa-star"></i>
+                                    <i class="fas fa-star"></i>
+                                </div>
+                                <div class="price-box mt-1 mr-2">
+                                    <span class="price text-red-price mr-2 f-16">475.000 đ</span>
+                                    <span class="compare-price f-14 text-normal">250.000 đ</span>
+                                </div>
+                           </div>
+
+                           <div class="product-description mt-3 pb-3 border-normal">
+                                <span class="f-13">Cây mọc thành bụi thưa, thân vươn thẳng với chiều cao cây trung bình từ 0,8-1,2m. Là hình trứng, mép lá nguyên, gân lá nổi rõ, phiến rộng khoảng từ 15-20cm. Lá có màu xanh đậm ở mặt trên, quanh gân lá có màu trắng sữa; mặt dưới lá màu nhạt hơn. Lá cây dạ lam mọc cách, cuống lá dài khi rụng để lại các khía màu nâu nhạt.</span>
+                           </div>
+
+                           <div class="product-quality mt-3 pb-3 border-normal">
+                                <div class="row d-flex align-items-center">
+                                    <label class="padding-left-15 text-dark mr-3 mb-0 d-flex align-items-center" data-translate="buyQuality"></label>
+                                    <button class="bg-white border-button-normal text-normal">-</button>
+                                    <div class="mr-2 ml-2 pl-5 pr-5 border-button-normal d-flex align-items-center">1</div>
+                                    <button class="bg-white border-button-normal text-normal">+</button>
+                                </div>
+                           </div>
+
+                           <div class="product-button d-flex justify-content-start mt-2">
+                                <button class="btn-buy firstb btn-cart button_35 left-to muangay add_to_cart mr-3" title="mua ngay" data-translate="productFeature.common.buyNow"></button>
+                                <a title="Xem nhanh" href="/cay-ngoc-ngan" data-handle="cay-ngoc-ngan" class="xem_nhanh btn-circle btn_view btn right-to quick-view hidden-xs hidden-sm border-normal mr-3">
+                                    <i class="fas fa-search"></i>
+                                </a>
+                                <a title="Yêu thích" class="xem_nhanh btn-circle btn_view btn right-to hidden-xs hidden-sm border-normal">
```

Comment:
```suggestion
                                <a title="Yêu thích" class="quick-view btn-circle btn_view btn right-to hidden-xs hidden-sm border-normal">
```
Xem_nhanh này a nhớ a cmt rồi mà

Reply:
Đợt trước anh cmt ở trang product ạ mà khi đó em làm trang này rồi em có lấy lại code bên đó nên em quên check lại ạ